liquibase-integration-tests/src/test/java/liquibase/dbtest/sqlite/SQLiteIntegrationTest.java

```diff
@@ -86,6 +88,7 @@ public void testOutputChangeLog() throws Exception {
     @Override
     public void tearDown() throws Exception {
         super.tearDown();
+        DatabaseObjectFactory.getInstance().reset();
```

Comment:
Should this go somewhere inside the AbstractIntegrationTest? Or wherever we run dropall, which I think is in the interceptor class?